firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -115,11 +118,17 @@ public static FirebaseInstallations getInstance(@NonNull FirebaseApp app) {
         .onSuccessTask(this::registerFidIfNecessary);
   }
 
-  /** Returns a auth token(public key) of this Firebase app installation. */
+  /**
+   * Returns a valid authentication token for the Firebase installation. Generates a new token if
+   * one doesn't exist, is expired or about to expire.
+   *
+   * <p>Should only be called if the Firebase Installation is registered.
+   */
   @NonNull
   @Override
-  public Task<InstallationTokenResult> getAuthToken(boolean forceRefresh) {
-    return Tasks.forResult(InstallationTokenResult.builder().build());
+  public Task<String> getAuthToken(final boolean forceRefresh) {
```

Comment:
doesn't make sense to mark a primitive parameter final

Reply:
You are right. That was by mistake. I forgot to remove it. Thanks for catching this.